tests/Predis/Command/Redis/FUNCTIONS_Test.php

```diff
@@ -178,11 +275,39 @@ public function testDeleteFunctionRemovesAlreadyExistingLibrary(): void
     public function testDeleteFunctionThrowsErrorOnNonExistingLibrary(): void
     {
         $redis = $this->getClient();
-        $redis->executeRaw(['FUNCTION', 'FLUSH']);
+        $redis->function->flush();
 
         $this->expectException(ServerException::class);
         $this->expectExceptionMessage('ERR Library not found');
 
         $redis->function->delete($this->libName);
     }
+
+    public function flushArgumentsProvider(): array
+    {
+        return [
+            'with default arguments' => [
+                ['FLUSH', null],
+                ['FLUSH'],
+            ],
+            'with mode argument' => [
+                ['FLUSH', 'sync'],
```

Comment:
TBH I don't think you need to test the uncased version. I'd approve either way - but this is redis side. We should send our commands, and allow it to do the right thing.

Reply:
@chayim Gotcha! Won't do this again